src/OpenMPIR.h

```diff
@@ -730,6 +730,7 @@ class OpenMPDependClause : public OpenMPClause {
     static OpenMPClause * addDependClause(OpenMPDirective *, OpenMPDependClauseModifier, OpenMPDependClauseType);
     std::string toString();
     void generateDOT(std::ofstream&, int, int, std::string);
+    void mergeDepend(OpenMPDirective*, OpenMPClause*);
```

Comment:
Next step, consider using the virtual function ```normalize``` instead of this kind of clause for a particular clause.